src/core/Table/filters/DateRangeFilter/DatePickerInput.tsx

```diff
@@ -66,6 +66,7 @@ const DatePickerInput = (props: DatePickerInputProps) => {
           close();
         }
       }}
+      appendTo='parent'
```

Comment:
Why do some of these have `appendTo='parent'` now?

The submenu one is a known issue #542 but I'm surprised this date picker input breaks.

Reply:
Would need to catch events ourselves and check the target. There is prop `[hideOnClick](https://atomiks.github.io/tippyjs/v5/all-props/)`, which we could change to `toggle` and then just listen to clicks and check :/

Reply:
So wait, this `DatePickerInput` will always be used inside `ColumnFilter` which is already appended to body. Which means `appendTo='parent'` still places outside of the main component tree. Then I am ok with it actually

![image](https://user-images.githubusercontent.com/9084735/154513715-4097a92c-fc82-4da8-a2d0-a2cd5c61292b.png)

Reply:
Yes, exactly.
We will want to have a separate `DatePickerInput` one day, so then it should be important.